docs/config/README.md

```diff
@@ -488,6 +489,35 @@ The `line_break` module separates the prompt into two lines.
 disabled = true
 ```
 
+## Memory Usage
+
+The `memory_usage` module shows current system memory and swap usage.
+
+By default the swap usage is displayed if the total system swap is non-zero.
+
+### Options
+
+| Variable          | Default               | Description                                                   |
+| ----------------- | --------------------- | ------------------------------------------------------------- |
+| `show_percentage` | `false`               | Display memory usage as a percentage of the available memory. |
+| `show_swap`       | variable              | Display swap usage.                                           |
```

Comment:
Is `when above zero`, better than `variable` here?

Reply:
The logic is when total system swap is non-zero, so maybe `when total swap non-zero` works here.